<!DOCTYPE html>
<html lang="en">
<head>
    <title>Mesirov Lab</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link type="text/css" rel="stylesheet" href="../css/styles.css">
</head>
<body style="background-color: #F7F7FF;">
    <div style="margin: 0 auto;">
        <img src="../img/top-banner.png" style="width: 101%; margin-left: -10px; ">
    </div>
    <nav class="navbar">
        <div class="container-fluid">
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
            </div>
            <div class="collapse navbar-collapse" id="myNavbar">
                <ul class="nav navbar-nav" id="navigation">
                    <li><a href="../">HOME</a></li>
                    <li><a href="../about/">ABOUT</a></li>
                    <li><a href="../members/">PEOPLE</a></li>
                    <li><a href="../projects/">RESEARCH</a></li>
                    <li><a href="../pubs/">PUBLICATIONS</a></li>
                    <li><a href="../software/">SOFTWARE</a></li>
                    <li><a href="../teaching/">TEACHING</a></li>
                </ul>
            </div>
        </div>
    </nav>
    
    <!-- <div id="project-carousel" class="carousel">
        <div class="carousel-inner">
            <div class="item active">
                <img src="../img/gsea-two.png?x=3" alt="GSEA" width="900">
            </div>
        </div>
        <a class="left carousel-control" href="#project-carousel" data-slide="prev">
            <span class="glyphicon"></span>
        </a>
        <a class="right carousel-control" href="#project-carousel" data-slide="next">
            <span class="glyphicon"></span>
        </a>
    </div> --> 
    
    <div id="project-main" style="margin: 0 auto;">
        <div class="container" id="project-container">
            <div id="title">
                <h1>RESEARCH PROJECTS</h1>
                <hr>
            </div>
            <div>
                <h2>New statistical and machine learning techniques for cancer genomics:</h2>
                <p>Our lab has primarily focused on the application of machine learning and statistical techniques to find <b>molecular markers to aid in the diagnosis, prognosis, and treatment of cancer</b> as well as to understand the underlying biological processes. The early work used the behavior of individual genes as markers, or variables, to which algorithms are applied. In a series of collaborations, we successfully completed a number of the first classification or prediction projects to determine both morphology and treatment outcome in pretreatment cancer data sets. Subsequently, using a new Bayesian integrative approach and employing signatures and pathways rather than individual genes (see next section), we were able to develop a patient stratification predictive model that was the first to generalize to an independent cohort and data set. 
                </p>
                <button data-toggle="collapse" data-target="#project-pub1" class="btn btn-default">Relevant Publications</button>
                <a href="http://mesirovlab.org/medulloblastoma/" class="btn btn-default" style="margin-left: 5px; font-weight: 500;" role="button">Medulloblastoma Site</a>
                <div id="project-pub1" class="collapse" style="padding: 0px;">
                    <p>1. Golub TR, Slonim DK, Tamayo P, Huard C, Gaasenbeek M, <b>Mesirov JP</b>, Coller H, Loh ML, Downing JR, Caligiuri MA, Bloomfield CD, Lander ES. Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science. 1999 Oct 15;286(5439):531-7. PMID: 10521349</p>
                    <p>2. Pomeroy SL, Tamayo P, Gaasenbeek M, Sturla LM, Angelo M, McLaughlin ME, Kim JY, Goumnerova LC, Black PM, Lau C, Allen JC, Zagzag D, Olson JM, Curran T, Wetmore C, Biegel JA, Poggio T, Mukherjee S, Rifkin R, Califano A, Stolovitzky G, Louis DN, <b>Mesirov JP</b>, Lander ES, Golub TR. Prediction of central nervous system embryonal tumour outcome based on gene expression. Nature. 2002 Jan 24;415(6870):436-42. PMID: 11807556</p>
                    <p>3. Cho YJ, Tsherniak A, Tamayo P, Santagatax S, Ligon A, Greulich H, Lu J, Berhoukim R, Getz G, Amani V, Goumnerova L, Eberhart CG, Lau CC, Olson JM, Gilbertson RJ, Gajjar A, Haberler C, Delattre O, Kool M, Ligon K, Meyerson M, <b>Mesirov JP*</b>, Pomeroy SL*. Integrative genomic analysis of medulloblastoma identifies a molecular subgroup that drives poor clinical outcome. J Clin Oncol. 2011 Apr 10;29(11):1424-30. PMCID: PMC3082983</p>
                    <p>4. Tamayo PT, Cho YJ, Tsherniak A, Greulich H, Ambrogio L, Schouten-van Meeteren N, Zhou T, Buxton A, Kool M, Meyerson M, Pomeroy SL*, <b>Mesirov JP*</b>, Predicting relapse in medulloblastoma patients by integrating evidence from clinical and genomic features. J Clin Oncol. 2011 Apr 10;29(11):1415-23. PMCID: PMC3082982</p>
                </div>
            </div>
            <div>
                <h2>New statistical and mathematical techniques for interpreting genomic data:</h2>
                <p>From a biological perspective, we know it is not individual genes but rather sets of genes, representing pathways or states of the cell, that are the real features, by which we might predict the character of a particular patient sample. We made a major contribution in this area by the introduction of a statistical methodology called <b>Gene Set Enrichment Analysis</b> (GSEA), which is now standard practice for interpreting global transcription profiling data and has thousands of citations. This was the first of a number of signature-based approaches we developed and applied to study genomic data. More recently, we have introduced information theoretic approaches to help elucidate the genomic alterations associated with biological phenotypes and identify novel therapeutics to treat disease.</p>
                <button data-toggle="collapse" data-target="#project-pub2" class="btn btn-default">Relevant Publications</button>
                <div id="project-pub2" class="collapse" style="padding: 0px;">
                    <p>1. Subramanian A, Tamayo P, Mootha V, Mukherjee S, Ebert BL, Gillette MA, Pomeroy S, Golub TR, Lander ES, <b>Mesirov JP</b>. Gene Set Enrichment Analysis: A Knowledge-Based Approach for Interpreting Genome-wide Expression Profiles. Proc Natl Acad Sci U S A. 2005 Oct 25;102(43):15545-50. PMCID: PMC1239896</p>
                    <p>2. Tamayo P, Scanfeld D, Ebert B, Gillette M, Roberts C, <b>Mesirov JP</b>.  Metagene Projection for cross-platform, cross-species characterization of global transcriptional states. Proc Natl Acad Sci U S A. 2007 Apr 3;104(14):5959-64. PMCID: PMC1838404</p>
                    <p>3. Kim JW, Botvinnik OB, Abudayyeh O, Birger C, Rosenbluh J, Shrestha Y, Abazeed  ME, Hammerman PS, DiCara D, Konieczkowski DJ, Johannessen CM, Liberzon A, Alizad-Rahvar AR, Alexe G, Aguirre A, Ghandi M, Greulich H, Vazquez F, Weir BA, Van Allen EM, Tsherniak A, Shao DD, Zack TI, Noble M, Getz G, Beroukhim R, Garraway LA, Ardakani M, Romualdi C, Sales G, Barbie DA, Boehm JS, Hahn WC, <b>Mesirov JP</b>, Tamayo P. Characterizing genomic alterations in cancer by complementary functional associations. Nat Biotechnol. 2016 May;34(5):539-46. doi: 10.1038/nbt.3527. Epub 2016 Apr 18. PubMed PMID: 27088724; PMCID: PMC4868596.</p>
                    <p>4. Hanaford AR, Archer TC, Price A, Kahlert UD, Maciaczyk J, Nikkhah G, Kim JW, Ehrenberger T, Clemons PA, Dančík V, Seashore-Ludlow B, Viswanathan V, Stewart ML, Rees MG, Shamji A, Schreiber S, Fraenkel E, Pomeroy SL, <b>Mesirov JP</b>, Tamayo P, Eberhart CG, Raabe EH. DiSCoVERing Innovative Therapies for Rare Tumors: Combining Genetically Accurate Disease Models with In Silico Analysis to Identify Novel Therapeutic Targets. Clin Cancer Res. 2016 Aug 1;22(15):3903-14. PMID: 27012813.</p>
                </div>
            </div>
            <div>
                <h2>Identifying states related to infectious disease:</h2>
                <p>We used our GSEA and metagene projection methods to study the in vivo transcriptional profiles of the malaria parasite Plasmodium falciparum, i.e., from samples taken from the human hosts. The computational methods enabled us to use signatures of known states in yeast (Saccharomyces cerevisiae) to identify previously unknown physiological diversity in P. falciparum. The work was highly novel in both its findings and the methodology we employed. In a more recent project we used our GSEA methods and a signature based approach on pre-vaccination molecular profiles to identify vaccinees that will have a strong vs. weak immune response.
                </p>
                <button data-toggle="collapse" data-target="#project-pub3" class="btn btn-default">Relevant Publications</button>
                <div id="project-pub3" class="collapse" style="padding: 0px;">
                    <p>1. Daily JP, Scanfeld D, Pochet N, Le Roch4 K, Plouffe D, Kamal M, Sarr O, Mboup S, Ndir O, Wypij D, Levasseur K, Thomas E, Tamayo P, Dong C, Zhou Y, Lander ES, Ndiaye D, Wirth D, Winzeler E, <b>Mesirov JP</b>, Regev A. Distinct physiological states of the parasite Plasmodium falciparum in malaria infected patients. Nature. 2007 Dec 13;450(7172):1091-5. PMID: 18046333</p>
                    <p>2. Tan Y, Tamayo P, Nakaya H, Pulendran B, <b>Mesirov JP</b>, Haining WN. Gene signatures related to B-cell proliferation predict influenza vaccine-induced antibody response. Eur J Immunol. 2014 Jan;44(1):285-95. PMCID: PMC3973429</p>
                </div>
            </div>
            <div>
                <h2>Software and reproducible research:</h2>
                <p>Importantly, our lab is dedicated to disseminating the mathematical and computational methods we develop as accessible software tools. These tools serve in aggregate over 300,000 investigators world-wide and include the GSEA software package and our comprehensive repository of gene sets, the <b>Molecular Signatures Database </b>(MSigDB) used with GSEA to interpret genomic data; an integrative computational genomics software environment, <b>GenePattern</b>; the <b>Integrative Genomics Viewer</b> (IGV), a high-performance visualization tool for interactive exploration of large, integrated genomic datasets; and the <b>GenomeSpace</b> platform which supports transparent interoperability between popular bioinformatics tools and data resources.</p>
                <button data-toggle="collapse" data-target="#project-pub4" class="btn btn-default">Relevant Publications</button>
                <a href="../software/" class="btn btn-default" style="margin-left: 5px; font-weight: 500;" role="button">Learn more...</a>
                <div id="project-pub4" class="collapse" style="padding: 0px;">
                    <p>1. Reich M, Liefeld T, Gould J, Lerner J, Tamayo P, <b>Mesirov JP</b>. GenePattern 2.0. Nat Genet. 2006 May;38(5):500-1. PMID: 16642009</p>
                    <p>2. Robinson JT, Thorvaldsdóttir H, Winckler W, Guttman M, Lander ES, Getz G, <b>Mesirov JP</b>. Integrative Genomics Viewer. Nat Biotechnol. 2011 Jan;29(1):24-6. PMCID: PMC3346182</p>
                    <p>3. Liberzon A, Subramanian A, Pinchback R, Thorvaldsdóttir H, Tamayo P, <b>Mesirov JP</b>. Molecular Signatures Database (MSigDB) 3.0 Bioinformatics. 2011 Jun 15;27(12):1739-40. PMCID: PMC3106198</p>
                    <p>4. Qu K, Garamszegi S, Wu F, Thorvaldsdottir H, Liefeld T, Ocana M, Borges-Rivera D, Pochet N, Robinson JT, Demchak B, Hull T, Ben-Artzi G, Blankenberg D, Barber GP, Lee BT, Kuhn RM, Nekrutenko A, Segal E, Ideker T, Reich M, Regev A, Chang HY, <b>Mesirov JP</b>. Integrative genomic analysis by interoperation of bioinformatics tools in GenomeSpace. Nat Methods. 2016 Mar;13(3):245-7. PMCID: PMC4767623. </p>
                </div>
            </div>
            <!--<div>
                <h2>Mathematical and computational research in support of the genome project:</h2>
                <p>Dr. Mesirov built and led computational efforts required for the sequencing, assembly, and analysis of the human and mouse genomes. A wide range of mathematics and computational approaches were needed for this work, including statistics and probability, combinatorial and graph algorithms, hidden markov models, pattern discovery, etc. Interestingly, because of the high volume of data and analyses to be done, her previous work in parallel and high performance computing proved to be important in addressing these challenges.</p>
                <button data-toggle="collapse" data-target="#project-pub5" class="btn btn-default">Relevant Publications</button>
                <div id="project-pub5" class="collapse" style="padding: 0px;">
                    <p>1. International Human Genome Sequencing Consortium. Initial sequencing and analysis of the human genome. Nature. 2001 Feb 15;409(6822):860-921. Erratum in: Nature. 2001 Aug 2;412(6846):565. Nature 2001 Jun 7;411(6838):720. PMID: 11237011</p>
                    <p>2. Batzoglou S, Jaffe DB, Stanley K, Butler J, Gnerre S, Mauceli E, Berger B, <b>Mesirov JP</b>, Lander ES. ARACHNE: a whole-genome shotgun assembler. Genome Res. 2002 Jan;12(1):177-89. PMCID: PMC155255</p>
                </div>
                <p><b>Key Collaborators:</b></p>
            </div> -->
        </div>
    </div>
    
<footer class="container-fluid text-center">
    <p>Mesirov Lab | UC San Diego School of Medicine | 9500 Gilman Drive | La Jolla, CA 92093</p>
</footer>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
</body>  
</html>
